src/main/java/com/mongodb/kafka/connect/source/MongoCopyDataManager.java

```diff
@@ -61,6 +65,21 @@
  */
 class MongoCopyDataManager implements AutoCloseable {
   private static final Logger LOGGER = LoggerFactory.getLogger(MongoCopyDataManager.class);
+  private static final String NAMESPACE_FIELD = "__";
+  private static final byte[] NAMESPACE_BYTES = "ns".getBytes(StandardCharsets.UTF_8);
+
+  private static final String PIPELINE_TEMPLATE =
+      format(
+          "{$replaceRoot: "
+              + "{newRoot: {"
+              + "_id: {_id: '$_id', copyingData: true}, "
+              + "operationType: 'insert', "
+              + "'%s': {db: '%%s', coll: '%%s'}"
```

Comment:
I have multiple questions about using quotes in strings that we call JSON (these questions are relevant not only to this PR):
1. Why are names not always encompassed in double quotes (`"`)? Both [JSON](https://www.rfc-editor.org/rfc/rfc8259.html) and [extended JSON](https://github.com/stIncMale/specifications/blob/master/source/extended-json.rst) require this.
2. Why is the namespace field name in single quotes?
3. Why are values encompassed in single quotes instead of double quotes? This, again, seem to violate both specifications.

Reply:
Here I'm going for readability over correctness.  The Java driver is very flexible in parsing json (most json parsers are).  Strictly speaking single quotes are invalid JSON but they are supported by the java driver as well as unquoted strings.

The namespace field was quoted because some special characters need quoting (such as "."). I've updated as its not needed for "__".